src/block/tile/Furnace.php

```diff
@@ -47,12 +49,24 @@ abstract class Furnace extends Spawnable implements Container, Nameable{
 	public const TAG_BURN_TIME = "BurnTime";
 	public const TAG_COOK_TIME = "CookTime";
 	public const TAG_MAX_TIME = "MaxTime";
+	public const TAG_STORED_XP = "StoredXPInt";
+
+	public static function getXpDropForItem(Item $raw, int $count) : int{
```

Comment:
For a question of logic, this function should not be placed between constants and properties